rustls/src/tls13/mod.rs

```diff
@@ -87,22 +86,41 @@ impl fmt::Debug for Tls13CipherSuite {
 }
 
 /// Constructs the signature message specified in section 4.4.3 of RFC8446.
-pub(crate) fn construct_client_verify_message(handshake_hash: &hash::Output) -> Vec<u8> {
-    construct_verify_message(handshake_hash, b"TLS 1.3, client CertificateVerify\x00")
+pub(crate) fn construct_client_verify_message(handshake_hash: &hash::Output) -> VerifyMessage {
+    construct_verify_message(handshake_hash, CLIENT_CONSTANT)
 }
 
 /// Constructs the signature message specified in section 4.4.3 of RFC8446.
-pub(crate) fn construct_server_verify_message(handshake_hash: &hash::Output) -> Vec<u8> {
-    construct_verify_message(handshake_hash, b"TLS 1.3, server CertificateVerify\x00")
+pub(crate) fn construct_server_verify_message(handshake_hash: &hash::Output) -> VerifyMessage {
+    construct_verify_message(handshake_hash, SERVER_CONSTANT)
 }
 
 fn construct_verify_message(
     handshake_hash: &hash::Output,
-    context_string_with_0: &[u8],
-) -> Vec<u8> {
-    let mut msg = Vec::new();
-    msg.resize(64, 0x20u8);
-    msg.extend_from_slice(context_string_with_0);
-    msg.extend_from_slice(handshake_hash.as_ref());
-    msg
+    context_string_with_0: &[u8; 34],
+) -> VerifyMessage {
```

Comment:
This feels like a constructor for `VerifyMesage` now. I think it'd be more natural as `VerifyMessage::new()`.